lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php

```diff
@@ -63,6 +63,9 @@ protected function initialize(InputInterface $input, OutputInterface $output) :
             );
             $connectionLoader        = new ConfigurationFile((string) $input->getOption('db-configuration'));
             $this->dependencyFactory = DependencyFactory::fromConnection($configurationLoader, $connectionLoader);
+        } elseif (is_string($input->getOption('configuration'))) {
```

Comment:
Can you please call `$input->getOption('configuration')` once and use a variable?